code/modules/mob/living/say.dm

```diff
@@ -301,6 +301,7 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	var/image/I = image('icons/mob/talk.dmi', src, "[bubble_type][say_test(message)]", FLY_LAYER)
 	I.appearance_flags = APPEARANCE_UI_IGNORE_ALPHA
 	INVOKE_ASYNC(GLOBAL_PROC, /.proc/animate_speechbubble, I, speech_bubble_recipients, 30)
+	INVOKE_ASYNC(GLOBAL_PROC, /.proc/animate_chat, src, message, message_language, message_mode, speech_bubble_recipients, 50) // see chatheader.dm

```

Comment:
Doesn't having this at the `/mob/living` level mean silicons won't have runescape chat?

Reply:
![image](https://i.imgur.com/eyk0BJ6.png)

Reply:
I'm dumb, silicons are a `living` subtype. Ignore this.